GitUI/UserControls/RevisionGrid/Graph/RevisionGraph.cs

```diff
@@ -207,31 +229,34 @@ public void Add(GitRevision revision, int? insertScore = null)
             // if insertScore is set use that (the scores must be reserved)
             int nextScore = insertScore ?? ++_maxScore;
 
-            if (_nodeByObjectId.TryGetValue(revision.ObjectId, out RevisionGraphRevision revisionGraphRevision))
+            if (_incompleteNodeByObjectId.TryGetValue(revision.ObjectId, out RevisionGraphRevision revisionGraphRevision))
             {
                 // This revision was added earlier, but is now found in the log.
                 // Increase the score to the current maxScore to keep the order intact.
                 revisionGraphRevision.EnsureScoreIsAbove(nextScore);
+                revisionGraphRevision.GitRevision = revision;
```

Comment:
Will remove this assigment, done below (was accidentally kept when splitting the modified insert artificial for #11267